src/proc.ts

```diff
@@ -153,6 +153,8 @@ export function execute(command: string, args?: string[], outputConsumer?: Outpu
             resolve({ retc: 0, stdout: '', stderr: '' });
         });
     } else {
+        // Since we won't be sending anything to this process, close its stdin.
+        spawn_opts.stdio = ['ignore', 'pipe', 'pipe'];
```

Comment:
Do you know why 'pipe' is in the array twice?

Reply:
corresponding to [stdin, stdout, and stderr]